drv/stm32xx-sys/src/main.rs

```diff
@@ -662,9 +668,14 @@ impl idl::InOrderSysImpl for ServerImpl<'_> {
                 // actually matched things.
                 let mut used_bits = 0u32;
 
-                for (i, entry) in
-                    generated::EXTI_DISPATCH_TABLE.iter().enumerate()
-                {
+                for i in 0..16 {
+                    // TODO: this sure looks like it should be using
+                    // iter.enumerate, doesn't it? Unfortunately that's not
+                    // currently getting inlined by rustc, resulting in rather
+                    // silly code containing panics. This is significantly
+                    // smaller.
+                    let entry = &generated::EXTI_DISPATCH_TABLE[i];
```

Comment:
*laughs nervously in my branch that adds more code iterating over `EXTI_DISPATCH_TABLE`*

Reply:
S'cool, I've worked out how to make that code not panic too.